sympy/stats/tests/test_rv.py

```diff
@@ -107,6 +107,14 @@ def test_IndependentProductPSpace():
 def test_E():
     assert E(5) == 5
 
+def test_H():
+    X = Normal('X', 0, 1)
+    D = Die('D', sides = 4)
+    G = Geometric('G', 0.5)
+    assert str(H(X, X > 0)) == '-log(2)/2 + 1/2 + log(pi)/2'
+    assert simplify(H(D, D > 2)) == log(2)
```

Comment:
Why `simplify`?

Reply:
I have removed it. The tests are passing without it on my system. I just used it for safety. Thanks for asking. It wasn't required here.